plugins/janus_streaming.c

```diff
@@ -3116,6 +3119,7 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 			json_t *videopt = json_object_get(root, "videopt");
 			json_t *videortpmap = json_object_get(root, "videortpmap");
 			json_t *videofmtp = json_object_get(root, "videofmtp");
+			json_t *videobufferkf = json_object_get(root, "videobufferkf");
```

Comment:
You'll need to add "videobufferkf" to the definition of `rtsp_parameters` as well, or it won't be validated when the JSON is parsed.

Reply:
done